pkg/test/helm/helm.go

```diff
@@ -35,6 +36,21 @@ func New(kubeConfig string) *Helm {
 	}
 }
 
+// Add Repo adds the specified helm repo to use for installing the charts
+func (h *Helm) AddRepo(t framework.TestContext) {
+	command := fmt.Sprintf("helm repo add istio %s", t.Settings().HelmRepo)
```

Comment:
can we/should we just add `--repo HelmRepo` to each command? The current command will modify users global helm settings (and may conflict if they have `istio` defined. Especially DeleteRepo

Reply:
Makes sense. Will fix

Reply:
@howardjohn done